src/client/pages/Checkout/shared/CheckoutIntercomVariation.tsx

```diff
@@ -2,26 +2,33 @@ import React from 'react'
 import styled from '@emotion/styled'
 import { Intercom } from 'utils/Intercom'
 import { MEDIUM_SCREEN_MEDIA_QUERY } from 'utils/mediaQueries'
+import { useFeature } from 'utils/hooks/useFeature'
+import { Feature } from 'shared/clientConfig'
 const TextLinkContainer = styled.div({
   textAlign: 'center',
   marginBottom: '-0.75rem',
   [MEDIUM_SCREEN_MEDIA_QUERY]: {
     display: 'none',
   },
 })
-export const CheckoutIntercomVariation = () => (
-  <>
-    <Intercom.InjectedStyles
-      style={{
-        display: 'none',
-        [MEDIUM_SCREEN_MEDIA_QUERY]: {
-          display: 'block',
-          bottom: '87px!important',
-        },
-      }}
-    />
-    <TextLinkContainer>
-      <Intercom.TextLinkVariation />
-    </TextLinkContainer>
-  </>
-)
+export const CheckoutIntercomVariation = () => {
```

Comment:
Can we render this conditionally instead of returning null?

Reply:
hmm but then we need to add the check in a lot more places

I think this looked fine when testing it out - but it's a workaround